src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -27,6 +27,7 @@
 import com.cloudbees.jenkins.GitHubWebHook;
 import com.cloudbees.plugins.credentials.CredentialsNameProvider;
 import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
@stephenc  I wasn't aware, thanks for pointing it out.

 Is it fine if I just add a comment in the method clarifying it is intended for testing purposes?